### PR TITLE
Update engine_yard.sh

### PR DESCRIPTION
**Pull request by drewkit**
Work completed to resolve recent report of Engine Yard breakage.

### CODE REVIEW

**Comment by mlocher** [comment]
Why do we need to source the script? That should not be necessary and can have a unintended side effects on the build environment.

Plus it's a breaking change from what we currently offer, so we can't release this as is in any case.

I'd prefer looking into a solution that doesn't require us to change how the script is used (to still calling it and not sourcing it)

**Comment by drewkit** [comment]
@mlocher needed to source the script in an attempt to support an `rvm use` call (to unblock `gem install engineyard` issue).

Latest update now makes it possible to call `rvm` as a function within bash script.

**Review by mlocher — APPROVED** [review]
We know that we have RVM available in `${HOME}/.rvm` (and not in `/usr/local/rvm`), but this is 👍 

Let's get this to the user and see if it works for them before merging though (as this is a rather large change and we don't have tests in place unfortunately)

**Pull request merged**
